tests/contenttypes_tests/tests.py

```diff
@@ -18,7 +21,26 @@
 
 @override_settings(ROOT_URLCONF='contenttypes_tests.urls')
 class ContentTypesViewsTests(TestCase):
-    fixtures = ['testdata.json']
+
+    @classmethod
+    def setUpTestData(cls):
+        # don't use the manager because we want to update the existing instance
+        # rather than "create" a new one with pk=1
+        cls.site1 = Site(pk=1, domain='testserver', name='testserver')
+        cls.site1.save()
```

Comment:
I didn't necessarily want to update the existing one, I just want the site to exist with those details. I had `.create()` here and it was failing because another instance existed with that id. So I changed it to a raw save, which will insert or update based on what's already there. I feel this is ok.